README.md

```diff
@@ -1,52 +1,54 @@
+## What is NovusCore
+NovusCore is an MMO Engine.
+
+Currently NovusCore is developing a game/server solution where our main focus is to be feature complete with the most popular MMOs out there.
+Long term, we hope to support many more features.
+
+## The NovusCore Promise
+The project was made with the promise that we would always focus on reliability, redundancy, and performance over convenience. We achieve this through making use of experience, but also applying modern techniques and design patterns.
+
+The end-goal is to provide a game/server setup, capable of tackling all of the limitations set by the current standard. Solving those issues are complicated, but we start by applying a proper foundation for our architecture to allow for better flow of information(data) and performance.
+
+## NovusCore Discord
+The project has an official [Discord](https://discord.gg/gz6FMZa).
+You will find the developers to be active on the discord and always up for answering any questions you might have regarding the project. Despite NovusCore not currently being ready for production level use, we are always welcoming any users that want to try using it.
+
+## Dependencies
+* [OpenSSL 1.1.0](https://www.openssl.org/source/)
+* [Premake5](https://premake.github.io/)
+* [Vulkan 1.1 (or higher)](https://vulkan.lunarg.com/)
+
+## Libraries
+Here we include a honorable mention to all the libraries included directly into the source of NovusCore. You do not need to download these on your own, but we felt it was important to address these as without them, NovusCore would be a lot more time consuming to develop.
+* [Moodycamel ConcurrentQueue](https://github.com/cameron314/concurrentqueue)
```

Comment:
These dependencies are a bit outdated, please use these ones:

* [All Dependencies from Engine](https://github.com/novusengine/Engine)
* [Jolt](https://github.com/jrouwe/JoltPhysics)
* [Casc](https://github.com/heksesang/CascLib)
* [Cuttlefish](https://github.com/akb825/Cuttlefish)

Reply:
Comment updated to properly refer to them as Libraries and not Dependencies, my bad!